### ISSUE TITLE
Fix heading nesting for Context section

### ISSUE THREAD

**Issue by grinnellm** [grinnellm/csas-style-sr#5]
The "Context" section fails Web Accessibility, due to inappropriate nesting. From the MS Word template, the title should be 'Heading 1', sections should be 'Heading 2', subsections should be 'Heading 3', and so on.
